### DISCUSSION TITLE
Installation with having Custom Theme in Filament Panel

### DISCUSSION THREAD

**Discussion by viraljetani** [awcodes/filament-curator#323] [Q&A] [answered]
Is there any way we can install this plugin when we are not using Filament Custom Themes but a default FilamentPHP installation. 

Reason being I am having an existing FilamentPHP v3 project and installed the plugin but the Picker & other CSS aren't loading properly inside the panel. I tried setting up a custom panel theme as mentioned in the docs but that causes issues with my existing panel's css. 

Would appreciate any help or pointers. Cheers!

**Comment by awcodes** [comment, marked as answer]
Should be the same. You just import the css into your own file and add the views to your tailwind.config.js.

**Comment by viraljetani** [comment]
Thanks so much for your response. 
I gave it another shot by creating a custom filament theme and added the css file imports to theme.css. Possibly it was a path issue when I tried it first. 

Thanks for the great plugin. Really loving it so far!

Cheers!
